tests/robots/test_configuration.py

```diff
@@ -6,27 +7,43 @@
 from compas_fab.robots import to_degrees
 
 
+def test_bool():
+    config = Configuration(joint_values=[1, 2, 3], types=[Joint.REVOLUTE]*3, joint_names=['a', 'b', 'c'])
+    assert config
+
+    config = Configuration(joint_values=[1, 2, 3], types=[Joint.REVOLUTE]*3)
+    assert config
```

Comment:
A test for `False` would also make sense (assuming we agree that `Configuration(joint_values=[0, 0, 0])` is really intended to be assumed as `False` when evaluated as bool)

Reply:
Oh, no, `Configuration(joint_values=[0, 0, 0])` is not `False`.

Reply:
potentially `Configuration(joint_names=['a', 'b', 'c'])` is `False`.